clients/spark-extensions/src/test/java/org/projectnessie/spark/extensions/ITNessieStatements.java

```diff
@@ -326,27 +311,28 @@ void mergeReferences() throws NessieConflictException, NessieNotFoundException {
     List<Object[]> resultList = commitAndReturnLog(refName);
     sql("USE REFERENCE %s", refName);
     sql("MERGE BRANCH");
-    List<Object[]> result = sql("SHOW LOG %s", refName);
-    assertEquals(
-        "log",
-        result.stream().map(ITNessieStatements::convert).collect(Collectors.toList()),
-        resultList);
+    // here we are skipping commit time as its variable
+    assertThat(
+            sql("SHOW LOG %s", refName).stream()
+                .map(ITNessieStatements::convert)
+                .collect(Collectors.toList()))
+        .containsExactlyElementsOf(resultList);
     spark.sessionState().catalogManager().setCurrentCatalog(catalog);
   }
 
   @Test
   void showLogIn() throws NessieConflictException, NessieNotFoundException, AnalysisException {
     List<Object[]> resultList = commitAndReturnLog(refName);
-    List<Object[]> result = sql("SHOW LOG %s IN nessie", refName);
     // here we are skipping commit time as its variable
-    assertEquals(
-        "log",
-        result.stream().map(ITNessieStatements::convert).collect(Collectors.toList()),
-        resultList);
+    assertThat(
+            sql("SHOW LOG %s IN nessie", refName).stream()
+                .map(ITNessieStatements::convert)
+                .collect(Collectors.toList()))
+        .containsExactlyElementsOf(resultList);
 
     // test to ensure property map is correctly encoded by Spark
     spark.sql(String.format("SHOW LOG %s IN nessie", refName)).createTempView("nessie_log");
-    result =
+    List<Object[]> result =
```

Comment:
Did you mean to remove this `result` variable?

Reply:
yes I forgot to remove it